qa-tests-backend/src/test/groovy/NodeInventoryTest.groovy

```diff
@@ -15,24 +17,55 @@ class NodeInventoryTest extends BaseSpecification {
         BaseService.useBasicAuth()
 
         // Get cluster ID
-        clusterId = ClusterService.getClusterId()
+        assert ClusterService.getClusters().size() > 0, "There must be at least one secured cluster"
+        clusterId = ClusterService.getClusters().get(0).getId()
         assert clusterId
     }
 
     @Tag("BAT")
     def "Verify node inventories and their scans"() {
         given:
-        "given a list of nodes"
+        "given a non-empty list of nodes"
         List<Node> nodes = NodeService.getNodes()
+        assert nodes.size() > 0
+
+        when:
+        boolean nodeInventoryContainerAvailable =
+            orchestrator.containsDaemonSetContainer(Constants.STACKROX_NAMESPACE, "collector", "node-inventory")
+        if (nodeInventoryContainerAvailable) {
+            log.info("Setting collector.node-inventory ROX_NODE_SCANNING_MAX_INITIAL_WAIT to 1s")
+            orchestrator.updateDaemonSetEnv(Constants.STACKROX_NAMESPACE, "collector", "node-inventory",
```

Comment:
Does this need a `cleanup:` section? Leaving it at `1s` for the remainder of the tests seems like it could be problematic.

Reply:
Commenting also here additionally to Slack: when the pods are restarted (as a result of executing this command), the initial run has already happened and this setting will not be used again (unless something else restarts the pods). The successive scans happen once every 4 hours.

If we would reset it to 5 minutes, then the pods would restart again and we would have another scan in 5 minutes - there is no need for that here.

The initial scan delay of 5 mins (randomized with max 5 mins) was meant for customers with huge clusters to prevent all initial scans to happen at the same time. In the context of this test, it does not matter if we keep it at 1s.